### PR TITLE
Homework 1 completed

### PR DESCRIPTION
**Pull request by NikitaKuzminn**

### CODE REVIEW

File: .idea/checkstyle-idea.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (komyagin) [inline .idea/checkstyle-idea.xml:1]: The directory .idea should be added to the .gitignore file. And this directory mustn't be here

File: epamHomeworks.iml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (komyagin) [inline epamHomeworks.iml:1]: The files *.iml should be added to the .gitignore file. And this file mustn't be here

File: target/checkstyle-cachefile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#Sat Oct 16 02:00:44 MSK 2021
```
Comment (komyagin) [inline target/checkstyle-cachefile:1]: The directory target should be added to the .gitignore file. And this directory mustn't be here

**Review by komyagin — CHANGES_REQUESTED** [review]

File: .idea/checkstyle-idea.xml (hunk 1 above)
Comment (NikitaKuzminn) [reply .idea/checkstyle-idea.xml]: Fixed

File: target/checkstyle-cachefile (hunk 3 above)
Comment (NikitaKuzminn) [reply target/checkstyle-cachefile]: Fixed

**Pull request merged**